titus-api/src/main/java/com/netflix/titus/api/jobmanager/service/V3JobOperations.java

```diff
@@ -41,71 +39,73 @@
         TaskMigration,
     }
 
-    Observable<String> createJob(JobDescriptor<?> jobDescriptor);
+    Observable<String> createJob(JobDescriptor<?> jobDescriptor, CallMetadata callMetadata);
 
     /**
-     * @deprecated Use {@link #updateJobCapacityReactor(String, Capacity)}
+     * @deprecated Use {@link #updateJobCapacityReactor(String, Capacity, CallMetadata)}
      */
-    Observable<Void> updateJobCapacity(String jobId, Capacity capacity);
+    Observable<Void> updateJobCapacity(String jobId, Capacity capacity, CallMetadata callMetadata);
```

Comment:
Given that every create/update method is now receiving `CallMetadata` and either `jobId` or `jobDescriptor` should we wrap them together so we don't need to pass them separately everywhere?

Reply:
This would be applicable only to mutating operations. We could do this if there are more bundled parameters, but for now I would leave it as it is.